modules/admin-ui-picard/app/src/components/events/partials/ModalTabsAndPages/SeriesDetailsThemeTab.js

```diff
@@ -0,0 +1,99 @@
+            {formik => (
+                <>
+                    <div className="modal-content">
+                        <div className="modal-body">
+                            <Notifications context="not-corner"/>
+                            <div className="full-col">
+                                <div className="obj quick-actions">
+                                    <header>{t('CONFIGURATION.NAVIGATION.THEMES')}</header>
+                                    <div className="obj-container padded">
+                                        <ul>
+                                            <li>
+                                                <p>{t('EVENTS.SERIES.NEW.THEME.DESCRIPTION.TEXT')}</p>
+                                                {themeNames.length > 0 && (
+                                                    <p>
+                                                        <Field name="theme"
+                                                               as="select"
+                                                               style={{width: '100%'}}>
+                                                            <option value={theme}>{theme}</option>
+                                                            {themeNames.map((theme, key) => (
+                                                                <option value={theme.value}
+                                                                        key={key}>{theme.value}</option>
+                                                            ))}
+                                                        </Field>
+                                                    </p>
+                                                )}
+                                            </li>
+                                        </ul>
+                                    </div>
+                                    {formik.dirty && (
+                                        <>
+                                            {/* Render buttons for updating theme */}
+                                            <footer style={{padding: '15px'}}>
+                                                <button type="submit"
+                                                        onClick={() => formik.handleSubmit()}
+                                                        disabled={!checkValidity(formik)}
+                                                        className={cn("submit",
+                                                            {
+                                                                active: checkValidity(formik),
+                                                                inactive: !checkValidity(formik)
+                                                            }
+                                                        )}>
+                                                    {t('EVENTS.SERIES.DETAILS.METADATA.REPLACE_SERIES_THEME')}
```

Comment:
I'm not shure, if this button works as it should. When trying it out, only the first time of changing the theme would work. When choosing a different theme afterwards and pressing the button again, nothing would happen.